### PR TITLE
Add tt styles 2

### PR DESCRIPTION
**Pull request by kabeaty**
#### What's the plan?

Add to The Texas Tribune sections so that each section at least includes some style info.
#### What's this PR do?

Adds styles to TT sections in markdown.
#### Why are we doing this? How does it help us?

Helps us add to the documentation of The Texas Tribune styles, which can be used across departments and help us prepare for the redesign this year.
#### How should this be manually tested?

Check out /texastribunestyles/ locally.
#### What are the relevant tickets?

Sprint task: https://basecamp.com/1866004/projects/1175549/todos/153506564
#### TODOs:
- [x] Add layout section
- [x] Add header section
- [x] Add footer section
- [x] Add widgets section
- [x] Add ads section
- [x] Add disclosure section
- [x] Update images section to multimedia instead and add to
- [x] Add to JavaScript section
- [x] Add feeds section
- [x] Add to buttons section
- [x] Add back in body class to get sidenav hover styles
- [x] Add more visual demos to sections

### CODE REVIEW

File: texastribunestyle.md [hunk 1]
```diff
@@ -121,29 +170,69 @@ to twelve. On mobile devices, each div with the class of `cell` becomes its own
 row.
 
 
+Responsiveness
+----
+
+The Texas Tribune uses four defined breakpoints -- 960px, 799px, 520px, and 320px. In addition, styles based on custom points are used throughout where needed for a design to look best at all sizes. A body class of `.responsive` should be added for responsiveness. This gives elements with the `.content-wrapper` class a width of 100% and max-width of the page width.
```
Comment (crccheck) [inline texastribunestyle.md]: I think we only have one breakpoint we actually use, mobile.

**Comment by crccheck** [comment]
You added a lot of documentation for things the site is currently doing, but only the buttons are backed by live demos.

**Comment by crccheck** [comment]
still an improvement over the original, so I'm ok with merging this. But I wish code that was documented in the styleguide was backed by a visual demo.

**Comment by kabeaty** [comment]
OK, I agree that's nice to have - I added some more visual demos to the page

**Pull request merged**